entrypoint.sh

```diff
@@ -94,6 +95,7 @@ then
     echo "resetting"
     rm -rf .git
     git init
+    git branch -m master ${archive_branch}
```

Comment:
If the image includes a recent enough git (2.28 or later), you could also run `git config --global init.defaultBranch "$archive_branch"` before `git init`. That would potentially future-proof this script in case git itself ever changes the default branch.

Reply:
This is a good idea! I'll change it